docs/modules/operation/pages/events/sources/eif-adapter.adoc

```diff
@@ -38,14 +38,15 @@ The feature has a debug level logging that can be used to debug operations.
 
```

Comment:
line 37 is a bit confusing.  "... has a debug logging level to debug operations."